static/scss/course-cards.scss

```
@@ -0,0 +1,105 @@
+// sass-lint:disable mixins-before-declarations
+.course-section h1 {
+  font-size: 44px;
+  font-weight: bold;
+  letter-spacing: 0;
+  line-height: 54px;
+  text-align: center;
+  padding-top: 54px;
+  padding-bottom: 54px;
+  margin: 0 auto;
+  max-width: 760px;
+
+  @include media-breakpoint-down(sm) {
+    font-size: 30px;
+    line-height: 40px;
+    padding-top: 20px;
+    padding-bottom: 22px;
+  }
```

This can be simplified. Our first thought should always be to match existing styles even if the design suggests something slightly different. It gives us less css to maintain, less chance of UI bugs, and we can easily change after a designer review

```suggestion
  text-align: center;
  padding: 50px 10%;
  margin: 0;

  @include media-breakpoint-down(sm) {
    font-size: 30px;
    padding: 20px 0;
  }
```